packages/core/src/actions/smartAccount/waitForUserOperationTransacation.ts

```diff
@@ -1,18 +1,14 @@
 import type { Chain, Hex, Transport } from "viem";
-import { getTransaction } from "viem/public";
-import type { SmartContractAccount } from "../../account/smartContractAccount.js";
+import { getTransaction } from "viem/actions";
 import type { BaseSmartAccountClient } from "../../client/smartAccountClient.js";
 import { Logger } from "../../logger.js";
 import type { WaitForUserOperationTxParameters } from "./types.js";
 
 export const waitForUserOperationTransaction: <
   TTransport extends Transport = Transport,
-  TChain extends Chain | undefined = Chain | undefined,
-  TAccount extends SmartContractAccount | undefined =
-    | SmartContractAccount
-    | undefined
+  TChain extends Chain | undefined = Chain | undefined
 >(
-  client: BaseSmartAccountClient<TTransport, TChain, TAccount>,
+  client: BaseSmartAccountClient<TTransport, TChain>,
```

Comment:
did this go from 3 to 2 generic types as part of the upgrade?

Reply:
ah we don't need the account here as a generic since waiting for a UO tx doesn't need account context